account/key/utils.go

```diff
@@ -0,0 +1,60 @@
+package key
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"errors"
+
+	sha256 "github.com/minio/sha256-simd"
+)
+
+func hashBytes(b1 []byte, b2 []byte) []byte {
+	h := sha256.New()
+	h.Write(b1)
+	h.Write(b2)
+	return h.Sum(nil)
+}
+
+func encrypt(base, key, data []byte) ([]byte, error) {
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return nil, err
+	}
+	// Never use more than 2^32 random nonces with a given key because of the risk of a repeat.
```

Comment:
You added this comment, but I don't understand what it has to do with the following line.

Reply:
I didn't add it. I've just copied.

Reply:
ok maybe we can improve it later then